clang/test/Driver/sycl-native-cpu.cpp

```diff
@@ -19,7 +19,14 @@
 // CHECK-WIN-NOT: dwarf
 
 // checks that -sycl-opt is not enabled by default on NativeCPU so that the full llvm optimization is enabled
-// RUN:   %clang -fsycl -fsycl-targets=native_cpu -### %s 2>&1 | FileCheck -check-prefix=CHECK-OPTS %s
+// Also check that we pass the expected backend options.
+// RUN:   %clang -fsycl -fsycl-targets=native_cpu -target aarch64-unknown-linux-gnu -march=armv9.4-a -### %s 2>&1 | FileCheck -check-prefix=CHECK-OPTS %s
```

Comment:
```suggestion
// RUN:   %clang -fsycl -fsycl-targets=native_cpu --target=aarch64-unknown-linux-gnu -march=armv9.4-a -### %s 2>&1 | FileCheck -check-prefix=CHECK-OPTS %s
```
Use of `-target <triple>` is supported, but it is considered legacy.

Reply:
I used this form to be consistent with the other check that's in this file; I'll update both to use `--target=`, thanks.

Reply:
There were a few more, all updated now, thanks!